src/xdocs/property_types.xml

```diff
@@ -925,9 +925,12 @@
       <p>
         This type represents a URI. The string representation is parsed using a custom
         routine to analyze what type of URI the string is.
-        It can be a URL, regular file, or a resource path. It will try loading the path as
-        a URL first, then as a file that must exist, and then finally as a resource on the
-        classpath.
+        It can be a URL, regular file, a file referenced using 'classpath:' protocol, or
+        a resource path. It will try loading the path as a URL first, then as a file that
+        must exist, and then finally as a resource on the classpath.
+
+        Note that the pseudo URL `classpath:` specifies that the resource
+        should be loaded from the class path.
```

Comment:
I would ensure this area and https://github.com/checkstyle/checkstyle/pull/9178#discussion_r567264881 reference that `classpath:` only loads from the java classpath if it isn't a file.

This is as discussed from conversation in https://github.com/checkstyle/checkstyle/pull/9178#discussion_r567265202 .

Reply:
> I would ensure this area ...

Doesn't the statement above this cover resource loading order?

`It will try loading the path as a URL first, then as a file that
        must exist, and then finally as a resource on the classpath.`

I have updated the javadoc for the method that you mentioned to specify order and give details on resource loading.

Reply:
Yes, that statement covers it, but this separate statement I highlighted here makes it seem like `classpath:` is an override, when it is just a last check. 

What about this?
````
        Note that the pseudo URL `classpath:` specifies that the resource
        should be loaded from the class path, if it is not a file.
````

Reply:
How about:

```
        Note that the pseudo URL `classpath:` specifies that the resource
        should be loaded from the class path, if it is not a local file.
```
?

Reply:
Good to me.

Reply:
Done.